app/models/user.rb

```diff
@@ -12,4 +12,8 @@ class User < ApplicationRecord
   def recent_posts
     posts.order(created_at: :desc).limit(3)
   end
+
+  def admin?
+    role == 'admin'
+  end
```

Comment:
- [OPTIONAL] Kudos for creating user.admin? method inside your user model. You can make it more expressive. 

```
  def admin?
      role == 'admin' ? true : false
  end
```

Reply:
alright @Qoosim, it will be made